modules/nf-core/rundbcan/cazymeannotation/tests/main.nf.test

```diff
@@ -0,0 +1,76 @@
+nextflow_process {
+
+    name "Test Process RUNDBCAN_CAZYMEANNOTATION"
+    script "../main.nf"
+    process "RUNDBCAN_CAZYMEANNOTATION"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "rundbcan"
+    tag "rundbcan/database"
+    tag "rundbcan/cazymeannotation"
+
+    test("dbcancazyme - simplified") {
+
+        setup {
+            run("RUNDBCAN_DATABASE"){
+                script "../../database/main.nf"
+                process {
+                    """
+                    """
+                }
+
+            }
+        }
+
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [id:'test_dbcan'],
+                    file(params.modules_testdata_base_path + 'genomics/prokaryotes/candidatus_portiera_aleyrodidarum/genome/proteome.fasta', checkIfExists: true)
+                ]
+                input[1] = RUNDBCAN_DATABASE.out.dbcan_db
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match()},
+                { assert path(process.out.cazyme_annotation.get(0).get(1)).exists() },
+                { assert path(process.out.dbcanhmm_results.get(0).get(1)).exists() },
+                { assert path(process.out.dbcansub_results.get(0).get(1)).exists()  },
+                { assert path(process.out.dbcandiamond_results.get(0).get(1)).exists()  }
+            )
+        }
```

Comment:
```suggestion
        then {
            assert process.success
            assertAll(
                { assert snapshot(
                    process.out,
                    path(process.out.versions[0]).yaml
                    ).match()
                }
            )
        }
```
Assuming all the output files are stable (i.e. the tool always gives the same, including on both docker/singularity and conda).